src/index.html

```diff
@@ -1026,6 +1026,23 @@ <h2 class="app__title">
 			</style>
 		</a>
 
+		<a class="list__item app js-app"
+				href="https://www.englishaccentsmap.com/"
+				data-app="english-accents-map"
+				data-tags="reference demo social other">
+		<div class="app__wrapper">
+				<h2 class="app__title">
+						English Accents Map
```

Comment:
Something is wrong with your indents. Could you please install [EditorConfig](http://editorconfig.org/) plugin to your editor, this will help you to use code style stated in our [.editorconfig](https://github.com/pwarocks/pwa.rocks/blob/master/.editorconfig).